inference-engine/src/mkldnn_plugin/mkldnn_graph.h

```diff
@@ -217,14 +217,21 @@ class MKLDNNGraph {
     void Allocate();
     void AllocateWithReuse();
     void CreatePrimitives();
+    void ExtractConstantNodes();
     void ExecuteConstantNodesOnly();
 
     friend class MKLDNNInferRequest;
     friend class MKLDNNGraphlessInferRequest;
     friend InferenceEngine::CNNNetwork dump_graph_as_ie_ngraph_net(const MKLDNNGraph &graph);
 
 private:
+    // these copies of graphNodes are to avoid regular checking for
```

Comment:
Pointers, not copies.

Reply:
Yes, thanks. But I left information about graphNodes because these ptrs are equal and connected